alphagov/paas-product-pages
# Update dependencies

Update of dependencies.

## How to test

- clone branch
- run `npm install`

To test pages in dev mode:
- run `npm run dev`
- check out pages look ok

To test production build:
- run `npm run dev`
-  `cd` to `out` directory
- serve pages with `npx serve`
- check out pages look ok
